ruby/BUILD.bazel

```diff
@@ -19,6 +21,46 @@ ruby_library(
     ],
 )
 
+java_export(
+    name = "protobuf_java_release",
+    maven_coordinates = "com.google.protobuf:protobuf-jruby:%s" % PROTOBUF_RUBY_VERSION,
+    srcs = glob(["src/main/java/com/google/protobuf/jruby/*"]),
+    runtime_deps = [
+        "@maven//:org_jruby_jruby_complete",
```

Comment:
You can change this to "@rules_ruby//ruby/runtime:jars" to select the system jruby jars

Reply:
Done.